### PR TITLE
fix: pass in root directory name where image file can be found

### PR DESCRIPTION
**Pull request by ericapisani**
The `__dirname` value used was referencing the compiled plugin's `__dirname` (e.g: `plugin/dist/...`) rather than the value we'd want which is the `__dirname` for the handler code running on AWS.

Also adds a Cypress test for the behaviour.

Test site: https://deploy-preview-113--gatsby-runner-hp.netlify.app/ (the logo should render correctly)

### CODE REVIEW

**Review by ascorbic — APPROVED** [review]
Looks good!

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (MarcL) [inline cypress/fixtures/example.json:5]: Is this file needed or just an artefact from Cypress install? Delete it if not?

Comment (ericapisani) [reply cypress/fixtures/example.json:5]: It's an artifact from the Cypress install. I left it in so the general file structure was there but if we'd rather not have it for now I can remove it

Comment (MarcL) [reply cypress/fixtures/example.json:5]: That's all good. We can leave it for future tests then. Thanks!
